packages/core/src/geometry/GeometrySystem.ts

```diff
@@ -313,12 +313,18 @@ export class GeometrySystem implements ISystem
      * @param {PIXI.Program} program - Instance of program
      * @param {boolean} [incRefCount=false] - Increment refCount of all geometry buffers
      */
-    protected initGeometryVao(geometry: Geometry, program: Program, incRefCount = true): WebGLVertexArrayObject
+    protected initGeometryVao(geometry: Geometry, shader: Shader, incRefCount = true): WebGLVertexArrayObject
```

Comment:
Can up update the doc above this to reflect this change?

Reply:
yessir